src/numerics.jl

```diff
@@ -0,0 +1,326 @@
+using UnPack
+using SparseArrays
+using LinearAlgebra
+
+include("Klimaparameter.jl")
+
+#Mapping functions
+##################
+function index2d(k,nx)
+    j = floor(Int,(k-1)/nx)+1
```

Comment:
Julia convention is to use spaces around operators and after commas, e.g., like so
```suggestion
    j = floor(Int, (k - 1) / nx) + 1
```